### DISCUSSION TITLE
dbt-core version in snowflake containers

### DISCUSSION THREAD

**Discussion by calleo** [dbt-labs/dbt-snowflake#750] [General]
Hi,

Running `docker run ghcr.io/dbt-labs/dbt-snowflake:1.5.2 --version` shows:

```
Core:
  - installed: 1.5.2
  - latest:    1.6.0 - Update available!

  Your version of dbt-core is out of date!
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation

Plugins:
  - snowflake: 1.5.2 - Update available!

  At least one plugin is out of date or incompatible with dbt-core.
  You can find instructions for upgrading here:
  https://docs.getdbt.com/docs/installation
```

As you see the `dbt-core` version is 1.5.2, but the current latest patch version is 1.5.4. What is the reasoning behind this? I think I would like to run the latest 1.5.x version of `dbt-core` and version 1.5.2 of `dbt-snowflake` when using the `1.5.2` image. Or am I missing something? Shouldn't I be doing this?

**Comment by Simon-Collinson** [comment]
Hi @calleo - I ran into a similar issue, ie, wanting to update dbt-core inside a docker image using the dbt-snowflake package.

It seems like the version number of dbt-snowflake is kept consistent with the version number of dbt-core. In other words, if you want to update to dbt-core v. 1.5.4, you will need to upgrade to dbt-snowflake v. 1.5.4 as well. In your case, `docker run ghcr.io/dbt-labs/dbt-snowflake:1.5.4 --version` should do the trick.

**Reply by Simon-Collinson** [reply]
Note to maintainers - as far as I can tell, the documentation doesn't cover this. It would probably be helpful to explicitly point out this version correspondence somewhere.

**Reply by aleenprd** [reply]
yeah no s***
